sdks/java/core/src/test/java/org/apache/beam/sdk/coders/CoderRegistryTest.java

```diff
@@ -74,28 +72,11 @@
   private static class NotSerializableClass { }
 
   @Test
-  public void testSerializableFallbackCoderProvider() throws Exception {
```

Comment:
There should still be an analogous test of falling back to `Serializable` for anything not registered.